docs/changelog.rst

```diff
@@ -6,6 +6,12 @@ Changelog
 `Unreleased`_
 -------------
 
+=======
+Added
+~~~~~
+
+- Shortcut for calling & validation. `#738`_
```

Comment:
Could you please add `.. _#738: https://github.com/schemathesis/schemathesis/issues/738` string to the end of this file where other links are placed? It will make the rendered changelog point to the proper issue location.